### PR TITLE
Fix Matic autoconfig, flesh out config defaults

### PR DESCRIPTION
**Pull request by samsondav**
- Add some defaults for Arbitrum
- Bump EthMaxQueuedTransactions for Matic
- Make sure matic config is actually used on mainnet
- Various other fixes

[ch11147]

### CODE REVIEW

File: core/store/orm/schema.go [hunk 1]
```diff
@@ -94,7 +94,7 @@ type ConfigSchema struct {
 	KeeperMaximumGracePeriod                  int64           `env:"KEEPER_MAXIMUM_GRACE_PERIOD" default:"100"`
 	KeeperMinimumRequiredConfirmations        uint64          `env:"KEEPER_MINIMUM_REQUIRED_CONFIRMATIONS" default:"12"`
 	KeeperRegistrySyncInterval                time.Duration   `env:"KEEPER_REGISTRY_SYNC_INTERVAL" default:"30m"`
-	LinkContractAddress                       string          `env:"LINK_CONTRACT_ADDRESS" default:"0x514910771AF9Ca656af840dff83E8264EcF986CA"`
+	LinkContractAddress                       string          `env:"LINK_CONTRACT_ADDRESS"`
```
Comment (samsondav) [inline core/store/orm/schema.go:97]: This is now set on a per-chain basis

File: core/store/orm/schema.go [hunk 2]
```diff
@@ -24,10 +24,10 @@ type ConfigSchema struct {
 	BridgeResponseURL                         url.URL         `env:"BRIDGE_RESPONSE_URL"`
 	ChainID                                   big.Int         `env:"ETH_CHAIN_ID" default:"1"`
 	ClientNodeURL                             string          `env:"CLIENT_NODE_URL" default:"http://localhost:6688"`
+	DatabaseBackupDir                         string          `env:"DATABASE_BACKUP_DIR" default:""`
```
Comment (samsondav) [inline core/store/orm/schema.go:27]: This just moved because it wasn't alphabetically ordered.

File: core/store/orm/config.go [hunk 3]
```diff
@@ -78,57 +78,84 @@ type (
 	// ChainSpecificDefaultSet us a list of defaults specific to a particular chain ID
 	ChainSpecificDefaultSet struct {
 		EnableLegacyJobPipeline          bool
+		EthBalanceMonitorBlockDelay      uint16
```
Comment (samsondav) [inline core/store/orm/config.go:81]: These are now in alphabetical order

File: core/store/orm/config_test.go [hunk 4]
```diff
@@ -56,17 +56,17 @@ func TestConfig_Profiles(t *testing.T) {
...
+		{"optimism", "420", 500000, 100000000000000},
 
-		{"bscMainnet", "56", 500000, 1000000000000000000},
-		{"hecoMainnet", "128", 500000, 1000000000000000000},
-		{"fantomMainnet", "250", 500000, 3500000000000000},
-		{"fantomTestnet", "4002", 500000, 3500000000000000},
-		{"polygonMatic", "800001", 500000, 1000000000000000000},
+		{"bscMainnet", "56", 500000, 100000000000000},
+		{"hecoMainnet", "128", 500000, 100000000000000},
+		{"fantomMainnet", "250", 500000, 100000000000000},
```
Comment (j16r) [inline core/store/orm/config_test.go:65]: Looks like you undid the change for fantom?

**Review by j16r — CHANGES_REQUESTED** [review]
Seems to have reverted fantom profile settings.

File: core/store/orm/config_test.go (hunk 4 above)
Comment (samsondav) [reply core/store/orm/config_test.go:65]: This was intentional - it turned out that Integrations misunderstood what this parameter did, and thought it was something else.

The correct values are:

1 LINK for mainnet
0.0001 LINK everywhere else

**Review by j16r — APPROVED** [review]

**Pull request merged**